support-payment-api/build.sbt

```diff
@@ -61,25 +59,7 @@ libraryDependencies ++= Seq(
   "org.scala-lang.modules" %% "scala-xml" % "1.2.0"
 )
 
-lazy val TeamCityTest = config("teamcity").extend(Test)
-
-enablePlugins(SystemdPlugin, PlayService, RoutesCompiler, RiffRaffArtifact, JDebPackaging, BuildInfoPlugin, GitVersioning)
-
-lazy val root = (project in file("."))
-  .configs(TeamCityTest)
-  .settings(inConfig(TeamCityTest)(Defaults.testTasks))
-  // Allows us to not run tests which require membership credentials in TeamCity - teamcity:test
-  // Stop gap until we decide how to handle integration tests.
-  .settings(
-    testOptions in TeamCityTest += Tests.Argument("-l", "tags.RequiresMembershipCredentials"),
-    buildInfoKeys := Seq[BuildInfoKey](
-      name,
-      BuildInfoKey.constant("gitCommitId", Option(System.getenv("BUILD_VCS_NUMBER"))
-        .getOrElse(git.gitHeadCommit.value.getOrElse("Unknown Head Commit")))
-    ),
-    buildInfoPackage := "app",
-    buildInfoOptions += BuildInfoOption.ToMap
-  )
+enablePlugins(SystemdPlugin, PlayService, RoutesCompiler, RiffRaffArtifact, JDebPackaging, BuildInfoPlugin)
```

Comment:
I think in the other projects, the enable plugins happens in the root build.sbt.  I am not sure which is better, but I thought I'd flag it in case you have an opinion.

Reply:
It looks like you are enabling/disabling some in the root and some here. It might be better to be consistent (do it all in the root?)

Reply:
👍 I've moved it to the root build.sbt